### PR TITLE
mighttpd2 3.2.9+fec3b11

### LINKED ISSUE #43924: mighttpd2 failed to build on 10.11
https://gist.github.com/1b06e73e2f548dafa610

```
Program/Mighty/Report.hs:23:8:
    Could not find module ‘Network.Wai.Handler.Warp.Timeout’
    it is a hidden module in the package ‘warp-3.1.2@warp_Bltc4LDQKgk2C2HEfpQREt’
    Use -v to see a list of the files searched for.
Failed to install mighttpd2-3.2.7
```

### PR DESCRIPTION
**Pull request by mietek**
Fixes #43924.

### CODE REVIEW

**Comment by mietek** [comment]
This will also fix one out of the two failures affecting [PR #47293](https://github.com/Homebrew/homebrew/pull/47293).

File: Library/Formula/mighttpd2.rb [hunk 1]
```diff
@@ -5,8 +5,9 @@ class Mighttpd2 < Formula
 
   desc "HTTP server"
   homepage "http://www.mew.org/~kazu/proj/mighttpd/en/"
-  url "https://hackage.haskell.org/package/mighttpd2-3.2.7/mighttpd2-3.2.7.tar.gz"
-  sha256 "57974d96b4dc5d8414ae61bcc45df4a0f07a855764a3baa57bdd39d0173c2dd0"
+  url "https://github.com/kazu-yamamoto/mighttpd2.git",
+    :revision => "fec3b111e26da3d7d9fcee3a7aeae53c0a26e597"
+  version "3.2.9+fec3b11"
```
Comment (mikemcquaid) [inline Library/Formula/mighttpd2.rb]: This needs to be a stable, released version for us to include it, unfortunately.

Comment (mietek) [inline Library/Formula/mighttpd2.rb]: Please do note the currently included version does not build. (https://github.com/Homebrew/homebrew/issues/43924)

Comment (mikemcquaid) [inline Library/Formula/mighttpd2.rb]: I know, that's annoying. Can you encourage upstream to make a new stable release?

Comment (mietek) [inline Library/Formula/mighttpd2.rb]: I can try. (https://github.com/kazu-yamamoto/mighttpd2/issues/15)

**Pull request closed without merging**